### PR TITLE
Add Cytomining organization profile readme and default Code of Conduct. 

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds an organizational profile readme to be displayed with the Cytomining organization page and a default Code of Conduct community health file. There are other options available for this repository, but I thought these might be a good start!

Thanks in advance for any feedback and input you may have on this!

I referenced the following documentation which also might be helpful for understanding this PR (also provided in the primary repo readme):
- [GitHub Docs: Organizations - Customizing your organization's profile](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile)
- [GitHub Docs: Building Communities - Creating a default community health file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at emailing [cytodata.info@gmail.com](mailto:cytodata.info@gmail.com).
```
Comment (gwaybio) [inline CODE_OF_CONDUCT.md:40]: @shntnu - do you know where these emails go? Can you add me as an auto forward?

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Welcome to Cytomining!
+
+[Cytomining](https://github.com/cytomining) is a software ecosystem that maintains essential tools, workflows, and best practices for single-cell image-based microscopy research.
```
Comment (gwaybio) [inline profile/README.md]:
```suggestion
[Cytomining](https://github.com/cytomining) is a software ecosystem that maintains essential tools, workflows, and best practices for single-cell image-based profiling and microscopy data analysis research.
```

**Review by gwaybio — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (shntnu) [reply CODE_OF_CONDUCT.md:40]: This is a CytoData Society email. Please see https://docs.google.com/document/d/1z6qg3CeNQidAFfH9QtjYUaA8q7QxAgexIfrd7IwBgkE/edit?disco=AAAA8JPPP6A

File: profile/README.md (hunk 2 above)
Comment (d33bs) [reply profile/README.md]: 👍 thanks, updating!

**Comment by d33bs** [comment]
Thank you @gwaybio and @shntnu for your help with review! Merging this in.

**Pull request merged**